src/reuse/modules/util/browser.ts

```diff
@@ -417,6 +417,46 @@ export class Browser {
 
     return false;
   }
+
+  /**
+   * @function isMobile
+   * @memberOf util.browser
+   * @description Indicates a mobile session
+   * @returns {boolean} Return true if its a mobile session driver.
+   * @example await util.browser.isMobile();
+   */
+  async isMobile(): Promise<boolean> {
+    const vl = this.vlf.initLog(this.isMobile);
+    vl.log('Indicates is a mobile session? or browser session');
+    return browser.isMobile();
+  }
+
+   /**
+   * @function isAndroid
+   * @memberOf util.browser
+   * @description Indicates a mobile session
+   * @returns {boolean} Return true if its a Android session driver.
+   * @example await util.browser.isAndroid();
+   */
+   async isAndroid(): Promise<boolean> {
+    const vl = this.vlf.initLog(this.isAndroid);
+    vl.log('Indicates is a Android session? or iOS session');
+    return browser.isAndroid();
+  }
+
+  /**
+   * @function isIOS
+   * @memberOf util.browser
+   * @description Indicates an iOS session
+   * @returns {boolean} Return true if its a iOS session driver.
+   * @example await util.browser.isIOS();
+   */
+  async isIOS(): Promise<boolean> {
```

Comment:
Would go with strict camelCase here - @cvakiitho @vlad9719 wdy think?

```suggestion
  async isIos(): Promise<boolean> {
```

Reply:
both are fine as for me